src/test/scala-2.11/markets/participants/OrderCancelerSpec.scala

```diff
@@ -17,22 +17,21 @@ package markets.participants
 
 import akka.actor.{ActorRef, ActorSystem}
 import akka.agent.Agent
-import akka.testkit.{TestProbe, TestActorRef, TestKit}
+import akka.testkit.{TestActorRef, TestKit, TestProbe}
 
-import java.util.UUID
-
-import markets.{Cancel, Canceled}
+import markets.{Cancel, Canceled, MarketsTestKit}
 import markets.orders.limit.LimitAskOrder
 import markets.tickers.Tick
-import markets.tradables.{Tradable, TestTradable}
-import org.scalatest.{Matchers, GivenWhenThen, FeatureSpecLike}
+import markets.tradables.{TestTradable, Tradable}
+import org.scalatest.{FeatureSpecLike, GivenWhenThen, Matchers}
 
-import scala.collection.{immutable, mutable}
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2385917872/issues/source?bid=2921361&fileBranchId=3228019#l28)